src/blk/kernel/KernelDevice.cc

```diff
@@ -540,10 +539,27 @@ void KernelDevice::_aio_stop()
 {
   if (aio) {
     dout(10) << __func__ << dendl;
+#if 1
+    IOContext wakeup_ctx(cct,nullptr,false);
+    wakeup_ctx.num_running = 1;
+    int pipe_fd[2];
+    if(pipe(pipe_fd) == -1){
+      perror("pipe");
```

Comment:
perror just prints error to stderr.
If we have an error there are 2 options:
a) ceph_assert
b) just skip waking up.

Since this is not mandatory function, only improvement, I think it should do b).